lib/rules/selector-not-notation/index.mjs

```diff
@@ -113,6 +114,7 @@ const rule = (primary) => {
 					node: ruleNode,
 					result,
 					ruleName,
+					line: range.start.line,
```

Comment:
The current order in `report.mjs` is

```js
	const range = node?.rangeBy({ index, endIndex }) ?? {};
	const startLine = line ?? range.start?.line;
```

we would need to change it to

```js
	const startLine = line ? line : node?.rangeBy({ index, endIndex })?.start?.line;
```

for it to matter.

Reply:
That's a good point. Fixed it.
[chore: call rangeBy only when needed](https://github.com/stylelint/stylelint/pull/8074/commits/21f1c034192812db511580f94c20d2d1e2dea59b)

Reply:
That was a just a remark; I should have said so, my bad.

Since it's a refactor, it's all good though.